controller/suite_test.go

```diff
@@ -74,15 +74,13 @@ var _ = BeforeSuite(func() {
 	Expect(err).NotTo(HaveOccurred())
 	Expect(cfg).NotTo(BeNil())
 
-	err = infrastructurev1alpha1.AddToScheme(scheme.Scheme)
-	Expect(err).NotTo(HaveOccurred())
+	Expect(infrastructurev1alpha1.AddToScheme(scheme.Scheme)).To(Succeed())
```

Comment:
nit: can we shorten this to `infrav1alpha1` for consistency with the other go files?